### ISSUE TITLE
Bug: does not handle non-ASCII subjects well

### ISSUE THREAD

**Issue by sebbASF** [apache/incubator-ponymail#443]
For example, the following mail has a mangled subject display:

https://lists.apache.org/thread.html/caf3f3f78c478020f0c0069715bd303f3ecb67bf169483a800e37ab9@%3Cdev.dubbo.apache.org%3E

The body text looks OK.

Note that the subject is handled OK by mod_mbox:
http://mail-archives.apache.org/mod_mbox/incubator-dubbo-dev/201803.mbox/%3c152240877614.5045.17528549688321645295.gitbox@gitbox.apache.org%3e

**Comment by Humbedooh** [comment]
I believe this isn't to do with Pony Mail, but rather that the subject hasn't been base64-encoded properly in the email. Sounds more like an infra issue.

**Comment by sebbASF** [comment]
However mod_mbox copes fine with the mail.

**Comment by Humbedooh** [comment]
That is true - doesn't mean pony mail has a bug just because mod_mbox is more lenient with faulty emails :). PM tries to decode headers, provided they actually are encoded.

I tried with a "proper" client, and that seems to work fine, https://lists.apache.org/thread.html/794eca46034fd8de91c9f4a6845c5480e8f7c82ccd6dc8af53935096@%3Cdev.ponymail.apache.org%3E

**Comment by sebbASF** [comment]
See also INFRA-16268
